roda-core/roda-core/src/main/java/org/roda/core/common/validation/ResourceResolver.java

```diff
@@ -40,10 +41,10 @@ public byte[] load(String href) throws Exception {
       try {
         String filename = href;
         try {
-          filename = Paths.get(URI.create(href)).getFileName().toString();
+          filename = FilenameUtils.normalize(Paths.get(URI.create(href)).getFileName().toString());
```

Comment:
This is a file name use FilenameUtils.getName

Reply:
Done